scout/adapter/mongo/case_events.py

```diff
@@ -147,7 +146,45 @@ def update_status(self, institute, case, user, status, link):
             {"$set": {"status": status}},
             return_document=pymongo.ReturnDocument.AFTER,
         )
-        LOG.debug("Case updated")
+
+        return updated_case
+
+    def tag_case(
+        self, institute: Dict, case: Dict, user: Dict, tags: List[str], link: str
+    ) -> Optional[dict]:
+        """Update the status tags of a case.
+
+        This function will create an Event to log that a user have updated the
+        status of a case. Also the status of the case will be updated.
+        Status tags could be any valid keys of the CASE_TAGS constant, eg
+            ("provisional", "diagnostic", "carrier", "upd", "medical attention")
+        """
+
+        for tag in tags:
+            if tag not in CASE_TAGS.keys():
+                LOG.warning("Tag {0} is invalid".format(tag))
+                return None
+
+        LOG.info(
```

Comment:
This won't be needed in production

Reply:
Hm, we do the same for the other events. I could change all?

Reply:
Yes, why not!